jetty-server/src/main/java/org/eclipse/jetty/server/session/Session.java

```diff
@@ -106,53 +107,39 @@
      * @see SessionCache
      *
      */
-    public class SessionInactivityTimeout extends IdleTimeout
+    public class SessionInactivityTimeout extends CyclicTimeout
```

Comment:
it is probably better for `SessionInactivityTimeout` to have-a `CyclicTimeout` rather that to be is-a.  That will avoid exposing the `schedule` method with arguments which is an alternative to `start`

Reply:
alternately, just make it a private inner class.